Resources/Locale/en-US/_NF/accent/goblin_accent.ftl

```diff
@@ -727,11 +777,61 @@ accent-goblin-words-replace-355 = touted
 accent-goblin-words-replace-356 = touts
 accent-goblin-words-replace-357 = toutin’
 accent-goblin-words-replace-358 = wo’ah
-accent-goblin-words-replace-359 = wever
+accent-goblin-words-replace-359 = wevah
 accent-goblin-words-replace-360 = wiv
 accent-goblin-words-replace-361 = wivout
 accent-goblin-words-replace-362 = screeve
 accent-goblin-words-replace-363 = screeves
 accent-goblin-words-replace-364 = screevin’
 accent-goblin-words-replace-365 = screeved
 accent-goblin-words-replace-366 = screeved
+accent-goblin-words-replace-367 = dis
+accent-goblin-words-replace-368 = dat
+accent-goblin-words-replace-369 = den
+accent-goblin-words-replace-370 = 'em
+accent-goblin-words-replace-371 = dey
+accent-goblin-words-replace-372 = dese
+accent-goblin-words-replace-373 = ya
+accent-goblin-words-replace-374 = dere
+accent-goblin-words-replace-375 = course
+accent-goblin-words-replace-376 = uvah
+accent-goblin-words-replace-377 = uvas
+accent-goblin-words-replace-378 = uverwise
+accent-goblin-words-replace-379 = yoo’re
+accent-goblin-words-replace-380 = yoo’ll
+accent-goblin-words-replace-381 = yoo’ve
+accent-goblin-words-replace-382 = yer
+accent-goblin-words-replace-383 = wot
+accent-goblin-words-replace-384 = 'ow
+accent-goblin-words-replace-385 = 'emselves
+accent-goblin-words-replace-386 = 'emself
+accent-goblin-words-replace-387 = froo
+accent-goblin-words-replace-388 = innit?
+accent-goblin-words-replace-389 = jus’
+accent-goblin-words-replace-390 = awake
```

Comment:
While I think I can see what you're going for here, "knows" to "awake" reads a bit oddly.

"You think he knows about this?" turning into "Ya fink 'e awake about dis?"
"Who knows?" turning into "Who awake?"

Not terrible, but a bit odd.

Speaking of, I do think you could specifically change "who" into "'oo", since it'll fail the H check.  If only you could differentiate an initial H from a W phoneme.